dds_web/scheduled_tasks.py

```diff
@@ -242,8 +242,42 @@ def monthly_usage():
             scheduler.app.logger.info(usage)
 
         scheduler.app.logger.debug("Task: Projects usage from database")
+        # a mock for a unit with two projects with some usage in bhours
+        usage_data_mock = {
```

Comment:
Seems like this is unused - is this intentional?

Reply:
it is intended for easier testing by only changing line 311

Reply:
Ah ok, fair enough. Could you comment it out and add a comment about this too? There's a risk that someone at some point removes it otherwise because the variable is unused.